Tests/State/InitStateTests.swift

```diff
@@ -0,0 +1,108 @@
+// The MIT License (MIT)
+//
+// ModernAVPlayer
+// Copyright (c) 2018 Raphael Ankierman <raphael.ankierman@radiofrance.com>
+//
+// InitStateSpecs.swift
+// Created by raphael ankierman on 28/02/2018.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import AVFoundation
+@testable
+import ModernAVPlayer
+import SwiftyMocky
+import XCTest
+
+final class InitStateTests: XCTestCase {
+    
+    private var context: PlayerContextMock!
+    private var contextDelegate: PlayerContextDelegateMock!
+    private var state: InitState!
+    private var player: MockCustomPlayer!
+    
+    override func setUp() {
+        ModernAVPlayerLogger.setup.domains = []
+        player = MockCustomPlayer()
+        context = PlayerContextMock()
+        contextDelegate = PlayerContextDelegateMock()
+        
+        Given(context, .player(getter: player))
+        Given(context, .delegate(getter: contextDelegate))
+        
+        state = InitState(context: context)
+    }
+
+    func testContextUpdatedCall() {
+        // ACT
+        state.contextUpdated()
+        
+        // ASSERT
+        // 2 because AVPlayer init set also this attribute
+        XCTAssertEqual(player.automaticallyWaitsToMinimizeStallingCallCount, 2)
```

Comment:
I would rather have used `XTCAssertEqualGreaterThanOrEqual(player.automaticallyWaitsToMinimizeStallingCallCount, 2)`
You don't care how much time your property is set, you only need to ensure it is done